test/jit_utils.py

```diff
@@ -90,13 +90,16 @@ def _isHookExceptionOk(self, e):
         return False
 
     def _compared_saved_loaded(self, m):
-        import zipfile
         # disable the hook while we parse code, otherwise we will re-enter the hook
         with torch.jit._disable_emit_hooks():
             try:
+                # short-circuit if this is an empty function or module
                 if len(m.code) == 0:
-                    # short-circuit if this is an empty module
                     return
+                if isinstance(m, torch._C.ScriptModule):
```

Comment:
Looks like some tests are just disabled, I am going to only review the non-test parts of this.